pkg/ceph/osd/agent_test.go

```diff
@@ -355,9 +355,12 @@ func TestGetPartitionPerfScheme(t *testing.T) {
 	assert.Equal(t, "sdc", scheme.Metadata.Device)
 	assert.Equal(t, 4, len(scheme.Metadata.Partitions))
 
-	// verify the first entry in the performance partition scheme.
+	// verify the first entry in the performance partition scheme.  note that the block device will either be sda or
+	// sdb because ordering of map traversal in golang isn't guaranteed.  Ensure that the first is either sda or sdb
+	// and that the second is the other one.
 	entry := scheme.Entries[0]
 	assert.Equal(t, 11, entry.ID)
+
 	// verify the first entry in the performance partition scheme.
```

Comment:
this comment is a dupe of the first sentence of the previous comment.  it can be removed i believe.

Reply:
ah i see the dup now